src/java/com/zimbra/cs/taglib/tag/LoginTag.java

```diff
@@ -266,4 +278,27 @@ public static void setTrustedCookie(HttpServletResponse response, String trusted
             ZimbraCookie.clearCookie(response, name);
         }
     }
+
+    public static boolean isCaptchaValid(String secretKey, String response) {
+        try {
+            String url = "https://www.google.com/recaptcha/api/siteverify?"
```

Comment:
Both secretKey and response should be URL encoded.

Reply:
done